server/cmd/main.go

```diff
@@ -38,7 +44,8 @@ var (
 
 const (
 	// DefaultProviderURL is the provider url for the "none" provider
-	DefaultProviderURL = "https://meshery.layer5.io"
+	DefaultProviderURL           = "https://meshery.layer5.io"
+	ArtifactHubComponentsHandler = "kubernetes" //The components generated in output directory will be handled by kubernetes
```

Comment:
Is this same "kubernetes" value being used when the Kubernetes components are registered or is "Meshery-server" used?

Does this variable correspond to host/registrant?

Reply:
Yes this is host. Yes this is the same for k8s components as well. Both are handed over to kubernetes.

Reply:
Ok. I thought that we had said `meshery-server` would be registered here, but now I recall how you're short-circuiting with a FK to the `kubernetes-context-id`.